plugins/modules/vmware_guest_disk.py

```diff
@@ -79,14 +79,19 @@
      - '     - C(thin) thin disk'
      - '     - C(eagerzeroedthick) eagerzeroedthick disk'
      - '     - C(thick) thick disk'
+     - '     - C(rdm) raw disk mapping'
```

Comment:
Add a note on line 73, stating that '`rdm` is added in Collection version 1.4.0'

Reply:
@Akasurde I've gone ahead and added the line you requested